.travis.yml

```diff
@@ -21,7 +21,7 @@ env:
     - secure: "AnnnXIFPy93JvGO+DuqSpYGPd9MZNSxwciYKTcCV6Yx5htJwZFU6adAak9UWvXr8vjtfwrHJD/VqViC6GII7toVCF9O9XEc6ooHVHsLGxfqvl0uOy+prgZS0F8bYW8n6EjFrWPfywdEoMn8PsZ/9iNxPFC2tH8vVKmJAq/FnxUo="
 
 script:
-  - RAILS_ENV=test bundle exec rake db:migrate --trace
+  - RAILS_ENV=test bundle exec rake db:schema:load
```

Comment:
@TheDudeWithTheThing what are your thoughts here?  

While I'm sure `db:schema:load` would be the faster way to run this,  the db:migrate runs pretty fast in travis already.   When we run a production deployment today, it's the migrations on the existing database that are running, not the db:schema:load.  So I care more about the migrations being always good than the full schema file.

Reply:
hmm, not bad I guess, but you need to run the migration locally first to get it into the schema file. At work we started loading just the schema since we have 4 years of migrations, if we can get away with running the migrations still I'm all for it.

Reply:
It's not really a speed thing, but as we just found out, you can run into trouble running migrations and avoid all problems loading schema.rb. It seems to make little sense (to me) running through the entire history of the migrations when you already have a snapshot of the outcome.

Reply:
when deploying in production, you're running the most recent migration(s) with the current version of Rails (low risk). But when running the entire migration history, you're running all migrations (maybe from years ago) with the current version of Rails (could be problematic).

Reply:
I'll take a CI problem any day of the week if it mitigates a production database migration problem.  Let's switch this back.

Reply:
happy to switch back to migrations iso schema load for CI, if you wish. However this does not increase the likelihood of catching production migration issues in CI (which would be indisputably good). 

A production migration problem (incremental migration) will be discovered in development. The "from-zero" migrations in CI simply introduce CI migration problems that do not pertain to development or production.